Tests/Unit/test_polymer_conformation.py

```diff
@@ -6,18 +6,42 @@
 
 from biocrnpyler import OrderedPolymerSpecies, PolymerConformation, Complex, Species
 
+def test_polymer_conformation_no_complex_instantiation():
+    p = OrderedPolymerSpecies([Species("m1"), Species("m2"), Species("m3")])
+    pc = PolymerConformation(polymer = p)
+    c1 = ComplexSpecies([p[0], p[1], Species("S")], called_from_complex = True)
+
+    assert str(pc) == str(p) #these should have the same name!
+    assert str(p) == str(pc.polymers[0])
+    assert len(pc.complexes) == 0
+
+    pc2 = PolymerConformation(polymer = [Species("m1"), Species("m2"), Species("m3")])
+    assert pc2 == pc
```

Comment:
interesting. Can you make them reverse here as well?

Reply:
What do you mean by that? that a polymer and its reverse are considered equal?

The reason for this is that, in order to keep code sane, I basically decided that either one would add complexes to a PolymerConformation or to a Polymer. This whole thing with binding a complex in a Conformation and updating the internal polymer lead to a ton of issues regarding the order of binding operations resulting in different names (I believe we had a good conversation about this a few months ago). Instead, I restricted the scope: if you want to use Conformations, the internal Polymer is immutable. However, that means a Conformation without anything should be the same as the underlying Polymer....